cardano-node/src/Cardano/CLI/Byron/UpdateProposal.hs

```diff
@@ -0,0 +1,192 @@
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Is `ScopedTypeVariables` necessary here? At a glance, it seems it could be removed.